src/app/_components/chats/ChatInput.tsx

```diff
@@ -119,26 +118,21 @@ export default function ChatInput({ chatId }: Props) {
             </FormItem>
           )}
         />
-        <div className="mr-3 flex flex-col justify-around">
+        <div className="flex items-center p-3 pt-0">
           <ImageUploadDialog
             onUploadSuccess={onUpload}
             uploadEndpoint="chatImageUploader"
             Button={
-              <Button
-                variant="secondary"
-                type="submit"
-                size="sm"
-                className="ml-auto gap-1.5"
-              >
-                upload image
-                <ImageUp className="size-3.5" />
+              <Button variant="outline" size="icon" type="submit">
+                <Image className="size-5 text-white" />
+                <span className="sr-only">Attach file</span>
               </Button>
             }
           />
 
-          <Button type="submit" size="lg" className="ml-auto w-full gap-1.5">
-            Send
-            <CornerDownLeft className="size-3.5" />
+          <Button type="submit" className="ml-auto gap-1.5">
+            Send Message
```

Comment:
for your consideration im not sure whats better, maybe dont use capital m on message

Reply:
ahlan